src/layouts/maintenance.hbs

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
(optional for now) if we will have a Japanese maintenance page, we can replace this with `<html lang={{#if (eq (site-profile) 'jp' )}}"jp"{{else}}"en"{{/if}}>`

Reply:
can you change this? It's probably good to do this now to make it future-proof.

After that, also make the same change for search-page.hbs please?